pkgs/development/python-modules/nlpo3/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  rustPlatform,
+  fetchFromGitHub,
+  pythonAtLeast,
+  rustc,
+  cargo,
+  setuptools,
+  setuptools-rust,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "nlpo3";
+  version = "1.3.0";
+  pyproject = true;
+
+  disabled = pythonAtLeast "3.12";
```

Comment:
Why disable python3.12?

Reply:
> Why disable python3.12?

because build failing on python 3.12... 
https://dpaste.org/TYauH

Reply:
@natsukium I think this can be fixed by updating the nlpo3's lockfile.

Upstream seems to be inactive, but we can try to upstream this change, and maybe *temporarily* merge this PR as is.

Or the updated Cargo.lock can probably be copied to nixpkgs, but I'm not sure if that's a good idea. It may also require some changes to the nlpo3 code.

What do you think about this?